ProcessRGB.cpp

```diff
@@ -3,6 +3,9 @@
 #include <limits>
 #ifdef __ARM_NEON
 #  include <arm_neon.h>
+#  ifdef __aarch64__
+#    define AARCH64_NEON
+#  endif
```

Comment:
I can also do `#if defined __ARM_NEON && defined __aarch64__` where relevant if that's preferred to adding a custom define. From a quick search I couldn't find a pre-existing compiler define for that combination, though maybe it exists.

Reply:
For consistency with rest of the code, please use the double-define check (you were not able to find it because it is two-staged elsewhere).

Reply:
Done.